src/main/java/io/vertx/openapi/contract/impl/OpenAPIContractImpl.java

```diff
@@ -65,8 +63,19 @@ public OpenAPIContractImpl(JsonObject resolvedSpec, OpenAPIVersion version, Sche
     this.rawContract = resolvedSpec;
     this.version = version;
     this.schemaRepository = schemaRepository;
+
     servers = unmodifiableList(resolvedSpec.getJsonArray(KEY_SERVERS, EMPTY_JSON_ARRAY).stream()
       .map(server -> new ServerImpl((JsonObject) server)).collect(toList()));
+
+    this.securityRequirements =
+      resolvedSpec.containsKey(KEY_SECURITY) ?
+        unmodifiableList(
+          resolvedSpec.getJsonArray(KEY_SECURITY).stream()
+            .map(o -> (JsonObject) o)
+            .map(SecurityRequirementImpl::new)
+            .collect(toList())) :
+        null;
```

Comment:
Is there a reason why returning null, or could we also return an empty list? An empty list is maybe less error prone.